src/components/Drawer/index.js

```diff
@@ -46,40 +47,49 @@ export default class Drawer extends React.Component {
     return header;
   };
 
+  getHeaderLayout = ({ toggleDrawer, isOpen }) => (
+    <HeaderContent>
+      <div>
+        {this.getHeaderContent({
+          toggleDrawer,
+          isOpen
+        })}
+      </div>
+      <CloseButton type="button" onClick={toggleDrawer}>
+        <HamburgerIcon
+          className={classNames({
+            hamburger: true,
+            "hamburger--opened": isOpen,
+            "hamburger--closed": !isOpen
+          })}
+        />
+      </CloseButton>
+    </HeaderContent>
+  );
+
   renderChildren = props => {
     const { children } = this.props;
     if (typeof children === "function") return children(props);
 
     return children;
   };
 
-  renderHeader = ({ toggleDrawer, isOpen }) => (
-    <Gradient
-      className="gradient--spotlight"
-      style={{ height: "60px", position: "relative" }}
-    >
-      <HeaderContent>
-        <div>
-          {this.getHeaderContent({
-            toggleDrawer,
-            isOpen
-          })}
-        </div>
-        <CloseButton type="button" onClick={toggleDrawer}>
-          <HamburgerIcon
-            className={classNames({
-              hamburger: true,
-              "hamburger--opened": isOpen,
-              "hamburger--closed": !isOpen
-            })}
-          />
-        </CloseButton>
-      </HeaderContent>
-    </Gradient>
-  );
+  renderHeader = ({ toggleDrawer, isOpen, withSpotLight }) => {
+    const headerLayout = this.getHeaderLayout({ toggleDrawer, isOpen });
+    return withSpotLight ? (
+      <Gradient
+        className="gradient--spotlight"
+        style={{ height: "60px", position: "relative" }}
+      >
+        {headerLayout}
+      </Gradient>
+    ) : (
+      headerLayout
+    );
+  };
 
   render() {
-    const { className, ...rest } = this.props;
+    const { className, withSpotLight, ...rest } = this.props;
```

Comment:
Looks good. Please add new tests for the `withSpotlight` prop and this will be ready for merge 👍

Reply:
Done